api/tests/functional-tests/backend/query/test_ops.py

```diff
@@ -1033,50 +1224,90 @@ def test_multiple_tables_in_args(
     ) in name_pairings
 
 
+def create_geospatial_inside_filter(
+    symbol_name: str,
+    value: Value,
```

Comment:
nit: `value` is very generic; consider making this code easier to follow by using a more specific name

Reply:
Its meant to be generic because it represents literally any value that is not symbolic.

Reply:
Do you have any suggestions?

Reply:
`inputted_value`, `user_input`, etc.; some other name which provides context about what this "value" means